subgraphs/burrow/src/handlers/actions.ts

```diff
@@ -361,7 +405,49 @@ export function handleBorrow(event: EventData): void {
   const usageHourlySnapshot = getOrCreateUsageMetricsHourlySnapshot(receipt);
   const financialDailySnapshot = getOrCreateFinancialDailySnapshot(receipt);
   const account = getOrCreateAccount(account_id.toString());
-  const position = getOrCreatePosition(account, market, "BORROWER", receipt);
+
+  // borrow position
+  const counterID = account.id
+    .concat("-")
+    .concat(market.id)
+    .concat("-")
+    .concat(PositionSide.BORROWER);
+  let positionCounter = _PositionCounter.load(counterID);
+  if (!positionCounter) {
+    positionCounter = new _PositionCounter(counterID);
+    positionCounter.nextCount = 0;
+    account.positionCount += 1;
+  }
+  const position = getOrCreatePosition(
+    account,
+    market,
+    PositionSide.BORROWER,
+    receipt,
+    positionCounter.nextCount
+  );
+
+  /**
+   * lending position
+   * @dev borrowed amount is deposited to lending position: it needs to be withdrawn to wallet to use
```

Comment:
This is interesting, can you point this out in the contract to me? I have never seen this